core-bundle/src/Resources/contao/dca/tl_module.php

```diff
@@ -185,6 +185,12 @@
 			'eval'                    => array('helpwizard'=>true, 'chosen'=>true, 'submitOnChange'=>true, 'tl_class'=>'w50'),
 			'sql'                     => "varchar(64) NOT NULL default 'navigation'"
 		),
+		'menuAlias' => array(
```

Comment:
If we remove the alias for now, we would also need to remove the Menu Provider, i.e. Twig integration. It would be a pity to not be able to fetch navigation modules in Twig templates, but I understand your concern for a more general alias concept in modules.

Reply:
It wasn't possible until now either. This PR consists of 2 features

- extending the navigation using events
- extended Twig support

Both have added values in their own right, sure. But in order to ease the review process we should keep the chunks as small as possible.

Reply:
Alias thingy removed from this PR and also see #3875